src/Commands/HelpController.php

```
@@ -0,0 +1,93 @@
+            $this->stderr('[scaffold] Module "scaffold" is not registered in this application.' . PHP_EOL);
+
+            return ExitCode::UNSPECIFIED_ERROR;
+        }
+
+        $files = glob($module->getControllerPath() . '/*Controller.php');
+
+        $rows = [];
+
+        if (is_array($files)) {
+            foreach ($files as $file) {
+                $id = Inflector::camel2id(substr(basename($file), 0, -14));
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\yii\helpers\Inflector' in method 'actionIndex'. (undefined)

(StaticAccess)